src/server/utils/container-manager.ts

```
@@ -1,30 +1,119 @@
+        type: "critical_server_error",
+        message: "Request Failed (container manager might be down)",
+      }
+
+    return {
+      status: "error",
+      type: "critical_server_error",
+      message: "Request Failed",
+    }
+  }
+
+  let json: any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)